src/engine-actions/get-peers.spec.js

```diff
@@ -0,0 +1,41 @@
+const path = require('path')
+const { expect, rewire, sinon } = require('test/test-helper')
+
+const getPeers = rewire(path.resolve(__dirname, 'get-peers'))
+
+describe('get-peers', () => {
+  let listPeersStub
+  let peers
+  let client
+
+  beforeEach(() => {
+    peers = [
+      { pubKey: '1234', address: '192.168.0.1:10009', inbound: false },
```

Comment:
do peers have an `inbound` property? What does it mean?

Reply:
@treygriffith they do. I added the field here to make sure that we are formatting peers to only include pubKey and address for now